flang/lib/Optimizer/Transforms/DebugTypeGenerator.cpp

```diff
@@ -57,6 +57,16 @@ DebugTypeGenerator::convertType(mlir::Type Ty, mlir::LLVM::DIFileAttr fileAttr,
                         mlir::StringAttr::get(context, logTy.getMnemonic()),
                         kindMapping.getLogicalBitsize(logTy.getFKind()),
                         llvm::dwarf::DW_ATE_boolean);
+  } else if (fir::isa_complex(Ty)) {
+    unsigned bitWidth;
```

Comment:
Would it make sense to initialize this to some value to avoid an uninitialized warning or alternatively add an else case below to assert or llvm unreachable?